### DISCUSSION TITLE
Custom icon from SVG file

### DISCUSSION THREAD

**Discussion by jfedyczak** [can-lehmann/owlkettle#175] [Q&A] [answered]
How can I import custom SVG and use it as icon?

**Comment by can-lehmann** [comment, marked as answer]
The `brew` procedure optionally takes an argument `icons` where you can pass a sequence of paths to folders containing icons. https://github.com/can-lehmann/owlkettle/blob/main/owlkettle.nim#L157

See for example: https://github.com/can-lehmann/owlkettle-crud/blob/main/src/main.nim#L83
